src/core/locator/qgslocatorfilter.cpp

```diff
@@ -44,6 +44,92 @@ bool QgsLocatorFilter::stringMatches( const QString &candidate, const QString &s
   return !search.isEmpty() && candidate.contains( search, Qt::CaseInsensitive );
 }
 
+double QgsLocatorFilter::fuzzyScore( const QString &candidate, const QString &search )
+{
+  QString candidateNormalized = candidate.simplified().normalized( QString:: NormalizationForm_C ).toLower();
```

Comment:
This should be calling QgsStringUtils::fuzzyScore directly -- it would avoid the duplicate code

Reply:
Oh, I completely forgot to cleanup there after moving to `QgsStringUtils`. My bad, sorry.